packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/order-summary-section.tsx

```diff
@@ -104,10 +110,54 @@ export const OrderSummarySection = ({ order }: OrderSummarySectionProps) => {
     (pc) => pc.status === "not_paid"
```

Comment:
question: Can we rely on this check? couldn't there be multiple unpaid collections?

Reply:
There's no way to get in this situation through the UI as of yet. So not possible right now. But when we link the payment collection to the order change, we can get rid of this check